README.md

```diff
@@ -25,154 +25,305 @@
+- Unit of measurement: Normalized composite score
+- Calculation: Weighted average of BIO_PM10, BIO_NO2, BIO_GreenBluePercentage, BIO_LandscapeQuality, and BIO_NDVI using theme-specific weights
+
+**QOL_Weighted**
+- Full name: Quality of Life Composite Score
+- Description: Weighted composite score of all quality of life metrics
+- Type of variable: Continuous (0-1)
+- Unit of measurement: Normalized composite score
+- Calculation: Weighted average of all QOL variables using theme-specific weights
+
+**CLI_Weighted**
+- Full name: Climate Adaptation Composite Score
+- Description: Weighted composite score of all climate adaptation metrics
+- Type of variable: Continuous (0-1)
+- Unit of measurement: Normalized composite score
+- Calculation: Weighted average of all CLI variables using theme-specific weights
+
+**Overall_Weighted**
+- Full name: Overall Composite Score
+- Description: Overall weighted composite score across all three themes
+- Type of variable: Continuous (0-1)
+- Unit of measurement: Normalized composite score
+- Calculation: Weighted combination of BIO_Weighted, QOL_Weighted, and CLI_Weighted scores
+
+Total number of variables: 21 (1 ID + 5 biodiversity + 6 quality of life + 5 climate adaptation + 4 composite scores)
+Total number of grid cells: 698
 
 # 5. SHARING/ACCESS INFORMATION
 ## 5.1 Licenses/restrictions placed on the data:
-...
+This dataset is licensed under Creative Commons Attribution-ShareAlike 4.0 International (CC BY-SA 4.0). You are free to share and adapt the material for any purpose, even commercially, under the following terms: you must give appropriate credit, provide a link to the license, and indicate if changes were made. If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.
 
 ## 5.2 Links to other resources:
 
 ### 5.2.1 Links to publications that cite or use the data:
-...
+To be updated upon publication
 
 ### 5.2.2 Links to other publicly accessible locations of the data: 
-...
+To be determined
 
 ### 5.2.3 Links/relationships to ancillary data sets: 
-...
+- Dresden Municipality WFS Endpoints (various environmental and urban planning datasets)
+- Copernicus Sentinel satellite imagery (NDVI data source)
+- ReBioClim project data repository: https://www.interreg-central.eu/projects/rebioclim/
 
 ### 5.2.4 Links to publicly accessible scripts for analysis of the dataset:
 - [Link title](link url)
 
 ### 5.2.5 Was data derived from another source?
-Yes/No
+Yes - Primary sources include Dresden municipality open data via WFS endpoints and Copernicus satellite imagery. All processing and analysis conducted by the research team.
 
 ## 5.3 Recommended citation for this dataset:
-...
+Schlosser, D., Bastiaansen, J., Chandrasekaran, A., & van Dijk, T. (2025). ReBioClim Dresden: Spatial Analysis Dataset for Urban Stream Restoration Prioritization. Delft University of Technology. [DOI to be assigned]
```

Comment:
When you start filling in the form on 4TU.ResearchData, you can reserve the DOI at the draft stage. Make sure you update the DOI here before uploading the file.